### PR TITLE
Switch SCD example to entity type `natural`

### PR DESCRIPTION
**Pull request by tlento**
## What are you changing in this pull request and why?

This changes the entity type for the SCD example to `natural`.

We recommend using the `natural` type for what is normally the `primary` entity in an SCD semantic model layout. The distinction is that `primary` indicates one row per entity value, while `natural` does not, and in an SCD context there is one row per entity value/time window set, rather than one row per entity value as in standard conformed dimension tables.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by tlento** [comment]
Oh wait, we might need to include the `primary_entity` designation here as well, let me see if I can update it. Or maybe we just leave this alone and let people overload `primary` in this way?

File: website/docs/docs/build/dimensions.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ This example shows how to create slowly changing dimensions (SCD) using a semant
 | 333             | 2    | 2020-08-19 | 2021-10-22| 
 | 333             | 3    | 2021-10-22 | 2048-01-01|  
 
-Take note of the extra arguments under `validity_params`: `is_start` and `is_end`. These arguments indicate the columns in the SCD table that contain the start and end dates for each tier (or beginning or ending timestamp column for a dimensional value).
+Take note of the extra arguments under `validity_params`: `is_start` and `is_end`. These arguments indicate the columns in the SCD table that contain the start and end dates for each tier (or beginning or ending timestamp column for a dimensional value). Note also the entity is tagged as `natural` - this is to distinguish it from a `primary` entity, where we expect one row per entity value. Here, we expect one row per entity value/validity window combination.
```
Comment (mirnawong1) [inline website/docs/docs/build/dimensions.md]:
```suggestion

The `validity_params` include two important arguments &mdash; `is_start` and `is_end`. These specify the columns in the SCD table that mark the start and end dates (or timestamps) for each tier or dimension. Additionally, the entity is tagged as `natural` to differentiate it from a `primary` entity. In a `primary` entity, each entity value has one row. In contrast, a `natural` entity has one row for each combination of entity value and its validity period.
```

**Comment by mirnawong1** [comment]
hey @tlento i left. sugg but let me know if it's best to not add it per eng requirements 🙏

File: website/docs/docs/build/dimensions.md [hunk 2]
```diff
@@ -282,7 +282,7 @@ semantic_models:
 
     entities:
```
Comment (tlento) [inline website/docs/docs/build/dimensions.md]:
```suggestion

    primary_entity: sales_person

    entities:
```

**Review by matthewshaver — APPROVED** [review]

**Comment by matthewshaver** [comment]
Feedback merged - shipping. Thank you @tlento !

**Pull request merged**